### DISCUSSION TITLE
How long my deployment will remain?

### DISCUSSION THREAD

**Discussion by Manjunadh-Velpuri** [vercel/community#4996] [Feature Question]
### Question

I have recently deployed my project on vercel. my project is completely on html, css and bootstrap. i have deployed it 3 days ago. how many days more i can expect my deployment to run? like, will it be deployed forever? is there any resource i can refer to know the deployments and other stuffs regarding the technology used?

**Comment by amyegan** [comment]
Hey @EndlessBrain! There is no set time limit, so your project will stay up until you delete it (unless it exceeds the [usage limits](https://vercel.com/docs/limits/overview) or violates our policies).

Here are some resources where you can learn more:
- [Vercel Documentation](https://vercel.com/docs)
- [Fair use Guidelines](https://vercel.com/docs/limits/fair-use-guidelines#)
- [Terms of Service](https://vercel.com/legal/terms)
- [Vercel Blog](https://vercel.com/blog)
